tests/standardise/test_standardise.py

```diff
@@ -551,3 +551,23 @@ def test_standardise_flux_timeseries():
     )
 
     assert "ch4_crf_uk" in flux_results
+
+def test_standardise_sorting():
```

Comment:
This test may need a few more checks if this is to check that the data has been stored in a sorted order - e.g. searching and retrieving the data with `sort=False` to ensure sorting isn't imposed when the data is returned.